fix: missing and misplaced periods

ibLeDy opened the pull request:
### Description
Fix inconsistent punctuation on `messages.py`

Missing periods
- Line 4
- Line 24
- Line 45
- Line 46
- Line 94
- Line 221
- Line 256
- Line 258
- Line 273
- Line 274

Misplaced periods
- Line 44

Fixes #248 

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)



### How Has This Been Tested?
Describe the tests you ran to verify your changes. Provide instructions or GIFs so we can reproduce. List any relevant details for your test.


### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged
- [ ] Update Postman API at /docs folder
- [ ] Update Swagger documentation and the exported file at /docs folder
- [ ] Update requirements.txt

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been published in downstream modules

ibLeDy committed:
fix: missing and misplaced periods

Fixes #248

Missing
- Line 4
- Line 24
- Line 45
- Line 46
- Line 94
- Line 221
- Line 256
- Line 258
- Line 273
- Line 274

Misplaced
- Line 44

isabelcosta commented on app/messages.py:
```diff
@@ -218,7 +218,7 @@
 }
 MENTORSHIP_RELATION_WAS_CANCELLED_SUCCESSFULLY = {
-    "message": "Mentorship" " relation was " "cancelled successfully"
+    "message": "Mentorship" " relation was " "cancelled successfully."
```
```suggestion
    "message": "Mentorship relation was cancelled successfully."
```

the double quotes between the whole message seem unnecessary 🤔 
I know that did not came from you, but it can easily be fixed here ;)

ibLeDy replied:
For sure, happy to do it!

isabelcosta reviewed (changes requested):
@ibLeDy you did everything right here. I just asked for a small change in a sentece you changed so its easy to spot and fix. Could you please fix it and commit again?

isabelcosta commented:
Also @ibLeDy thank you for following our guidelines 🙌 , following the PR template, the commit message guideline, referring the issue you were solving, waiting until assignment to send a PR.

ibLeDy committed:
fix: delete unnecessary double quotes

as suggested here: https://github.com/anitab-org/mentorship-backend/pull/498#discussion_r392703515

Co-Authored-By: Isabel Costa <isabelcmdcosta@gmail.com>

isabelcosta reviewed (approved):
Great work @ibLeDy 🎉 Thank you for your contribution!

ibLeDy commented:
Thanks, my pleasure!

ramitsawhney27 commented:
Thank you for your contribution. @isabelcosta looks good to merge.

isabelcosta commented:
@ramitsawhney27 thank you for reviewing and merging! 🎉  Could you please use the "Squash and Merge" option and squash commits while also making sure it follows the Commit Style guidelines of this project? I wrote [this wiki page](https://github.com/anitab-org/mentorship-backend/wiki/Maintainer-Guidelines) to show how this project has been maintained. I just want the git commit history to be clean so I can easily browse about the changes done to the code.
